test/integration/CMakeLists.txt

```diff
@@ -111,6 +112,7 @@ set(tests_needing_display
   shader_param_system.cc
   thermal_sensor_system.cc
   thermal_system.cc
+  triggered_camera.cc
```

Comment:
@iche033 This was removed from gz-sim7 for some reason. I added it back here, but if you know it should be removed, let me know.

Reply:
probably got lots in the forward ports. The test was forward ported [here](https://github.com/gazebosim/gz-sim/pull/1514/files#diff-612c1d43e6cd2b24070101fe026baf08ad423bbae381714d517a71d3c39495eb) but not the CMakeList.txt changes